e2e/pierced-react/src/routes/root.tsx

```diff
@@ -1,12 +1,10 @@
+import { Link } from "react-router-dom";
+
 function App() {
 	return (
 		<div className="index-page">
-			<a href="/remix-page" target="_blank" rel="noopener noreferrer">
-				Remix Page
-			</a>
-			<a href="/qwik-page" target="_blank" rel="noopener noreferrer">
-				Qwik Page
-			</a>
+			<Link to="/remix-page">Remix Page</Link>
+			<Link to="/qwik-page">Qwik Page</Link>
```

Comment:
I've used `a` with `target="_blank"` on purpose, so that here we do get hard navigations and the SSRed (from target) fragments are actually so (coming from the server/reframing from target)

if we were to have soft navigation here the "from target" fragments would become "from fragment if you do hard navigation and from fetch if you do soft navigation", isn't that more confusing? 😓 

is this really desirable?